### ISSUE TITLE
having problem trying to do multipart submission with zip file.

### ISSUE THREAD

**Issue by lhasadad** [Huachao/vscode-restclient#1055]
running into issue with an api I am using.   I can get the file to process and be recognized with Curl but having an issue with restclient.  I am sure it is something I am missing in how I formed the request:

```
###
# @name PostScheduledActions
POST {{baseUrl}}/scheduledActions/
Accept: application/json
Content-Type: multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW
Authorization: Bearer {{GetAuthToken.response.body.$.token}}

------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="replaceExisting"

true
------WebKitFormBoundary7MA4YWxkTrZu0gW
Content-Disposition: form-data; name="zipFile"
Content-Type: application/x-zip-compressed

< "H:\Git-Repos\cloudbolt-deployment-customizations\.vscode\truist_ldap_mapping.zip"
------WebKitFormBoundary7MA4YWxkTrZu0gW

```
 
Response I am getting:

```
HTTP/1.1 400 Bad Request
Date: Fri, 02 Sep 2022 19:20:11 GMT
Server: Apache
X-Frame-Options: SAMEORIGIN
X-Content-Type-Options: nosniff
Content-Length: 57
Vary: Accept,Cookie,Origin
Allow: GET, POST
Cache-Control: max-age=0
Content-Security-Policy: frame-ancestors 'self'
Connection: close
Content-Type: application/json


{
  "code": 400,
  "errors": [
    {
     "message": "zipFile is missing."
    }
  ]
}
```

The documentation for the API I am using is here:  https://app.swaggerhub.com/apis-docs/cloudbolt/Cloudbolt_CMP_API/2022.3.1#/Scheduled%20Actions/post_scheduledActions_

**Comment by MaximkaHuman** [comment]
Hi, your request is finished with invalid marker. Current `------WebKitFormBoundary7MA4YWxkTrZu0gW`
but should be `------WebKitFormBoundary7MA4YWxkTrZu0gW--`

**Comment by lhasadad** [comment]
I think I had that,  but will try again with that present.

**Comment by lhasadad** [comment]
With the extra '--' after the zip file it is still reporting the same problem.   If there is a way to get some info on what is going on (enhanced debug)  I can try and collect data to get better idea of what is being sent.

**Comment by MaximkaHuman** [comment]
Additionaly to my previous comment: check your path `"H:\Git-Repos\cloudbolt-deployment-customizations\.vscode\ldap_mapping.zip"` is correct. Problem can be in \, so check `"H:\\Git-Repos\\cloudbolt-deployment-customizations\\.vscode\\ldap_mapping.zip". As result: add -- to closed boundary and add second \ to any \ in your path, or replace \ by / in your path.

**Comment by lhasadad** [comment]
I note this but getting similar issue using this rather than fully qualified path.  I assume it checks both locations.  

You can also specify file path to use as a body, which starts with < , the file path(whitespaces should be preserved) can be either in absolute or relative(relative to workspace root or current http file) formats:

I tried it with thunder client and was able to get that to work and generate a block of code that looks like it will work too.

**Comment by lhasadad** [comment]
Just saw your prior note @MaximkaHuman  wand gave the escaped path  a try.   No luck.  I am sure i am doing something rather silly wrong or its just not possible (I tried full path with escaping and ./filename.zip as file name and had that in same directory as the file with the requests (.vscode) directory.

**Comment by MaximkaHuman** [comment]
@lhasadad Today I tested the latest version of REST Client and noticed that if path is wrapped by "" then multipart-form data file is not included into request body. Tested on Ubuntu 22.04.

**Comment by lhasadad** [comment]
@MaximkaHuman Will keep that in mind and try variations.   I do believe I have tried both ways and have tried with both relative path and absolute (windows) path (on windows instance of VS Code and REST Client)

**Comment by MaximkaHuman** [comment]
Full info can be taken from Wireshark. This log shows whats your requiest contains exacltly. If all filels of your requiest exist, but file body is empty, so the issue is in path format or wrong permission of file access.

**Comment by lhasadad** [comment]
@MaximkaHuman Good Idea to look at the network traffic with wireshark.  Do you also happen to know if there is a way to get REST Client to generate debug output (would be nice to know what it thinks its seeing in the request and what it is not)

**Comment by lhasadad** [comment]
@MaximkaHuman another option seems like it would be to pull a copy of the repo and debug the extension in Dev mode.
